spec/spec_helper.rb

```diff
@@ -34,5 +34,5 @@
 
 
 RSpec.configure do |config|
-
+  config.infer_spec_type_from_file_location!
```

Comment:
This is causing the build to fail when using Rspec < 3.0 . Can we either check for the Rspec version, or update to using Rspec > 3?